assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/AudienceItem.js

```diff
@@ -36,23 +36,37 @@ import {
 	AUDIENCE_SELECTION_FORM,
 } from './constants';
 import { CORE_FORMS } from '../../../../../../googlesitekit/datastore/forms/constants';
+import { CORE_USER } from '../../../../../../googlesitekit/datastore/user/constants';
 import { MODULES_ANALYTICS_4 } from '../../../../datastore/constants';
 import { numFmt } from '../../../../../../util';
+import NewBadge from '../../../../../../components/NewBadge';
 import { SelectionPanelItem } from '../../../../../../components/SelectionPanel';
 
+export const AUDIENCE_ITEM_NEW_BADGE_SLUG_PREFIX =
+	'audience-selection-panel-expirable-new-badge-';
```

Comment:
Since it's being used elsewhere, including the datastore, can we create it in `assets/js/modules/analytics-4/datastore/constants.js`?